.github/workflows/sonar.yaml

```diff
@@ -66,8 +66,8 @@ jobs:
           mvn -B clean test -Dcheckstyle.skip \
               -DenableTestCoverage \
               org.sonarsource.scanner.maven:sonar-maven-plugin:sonar \
-              -Dsonar.projectKey=mpeddada1_gapic-generator-java_unit_tests \
-              -Dsonar.organization=mpeddada1 \
+              -Dsonar.projectKey=googleapis_gapic-generator-java_unit_tests \
+              -Dsonar.organization=googleapis \
               -Dsonar.host.url=https://sonarcloud.io \
-              -Dsonar.projectName=showcase_unit_tests
+              -Dsonar.projectName=java_showcase_unit_tests
```

Comment:
Did you create this project in the googleapis organization? I don't see it there.

Reply:
Resolving this comment. Thanks for your help with creating the project in Sonarcloud @meltsufin!